<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRechargeDialog</class>
 <widget class="QDialog" name="CRechargeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>月租卡续费</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>起始日期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>终止日期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtEnd"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>增加天数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtDays">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>车辆类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxCarModel"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>收费金额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spAmount">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btnOk">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
